packages/zoe/src/contracts/sellItems.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-use-before-define */
+// @ts-check
+
+import harden from '@agoric/harden';
+import { makeZoeHelpers, defaultAcceptanceMsg } from '../contractSupport';
+
+/** @typedef {import('../zoe').ContractFacet} ContractFacet */
+
+// zcf is the Zoe Contract Facet, i.e. the contract-facing API of Zoe
+export const makeContract = harden(
+  /** @param {ContractFacet} zcf */ zcf => {
+    const { assertKeywords, rejectOffer, checkHook } = makeZoeHelpers(zcf);
+    assertKeywords(harden(['Items', 'Money']));
+
+    const { terms } = zcf.getInstanceRecord();
+    const { pricePerItem } = terms;
+
+    let sellerOfferHandle;
+
+    const sellerOfferHook = offerHandle => {
+      sellerOfferHandle = offerHandle;
+      return defaultAcceptanceMsg;
+    };
+
+    const buyerOfferHook = buyerOfferHandle => {
```

Comment:
Since this is touching the seller's state that is only available if another hook had gotten run, it should have a comment about why that's guaranteed to have happened first.  Is there a way to have that be structurally obvious (like the buyer hook is allocated in teh invocation of the seller hook or something).